source/auth/auth.md

```diff
@@ -1202,13 +1202,9 @@ in the MONGODB-OIDC specification, including sections or blocks that specificall
 #### [MongoCredential](#mongocredential) Properties
 
 > [!NOTE]
-> Drivers MUST NOT url-decode the entire `authMechanismProperties` given in an connection string when the
-> `authMechanism` is `MONGODB-OIDC`. This is because the `TOKEN_RESOURCE` itself will typically be a URL and may contain
-> a `,` character. The values of the individual `authMechanismProperties` MUST still be url-decoded when given as part
-> of the connection string, and MUST NOT be url-decoded when not given as part of the connection string, such as through
-> a `MongoClient` or `Credential` property. Drivers MUST parse the `TOKEN_RESOURCE` by splitting only on the first `:`
-> character. Drivers MUST document that users must url-encode `TOKEN_RESOURCE` when it is provided in the connection
-> string and it contains and of the special characters in \[`,`, `+`, `&`, `%`\].
+> Drivers MUST parse the `TOKEN_RESOURCE` by splitting only on the first `:` character. Drivers MUST document that users
```

Comment:
> Drivers MUST parse the `TOKEN_RESOURCE` by splitting only on the first `:` character

I'm unclear why TOKEN_RESOURCE gets special mention.  Does this require logic in parsers that is different for TOKEN_RESOURCE than any other auth mechanism property?

Reply:
This should be part of the connection string spec (I believe it's ambiguous there).

Reply:
I've moved the explanation to the connection string spec.